### ISSUE TITLE
v9.5.0 - Getting error after adding new backoffice language

### ISSUE THREAD

**Issue by alevak09** [umbraco/Umbraco-CMS#12384]
### Which *exact* Umbraco version are you using? For example: 9.0.1 - don't just write v9

9.5.0

### Bug summary

After adding a new croatian (hr.xml) language file in the folder umbraco/config/lang/ and loging into umbraco backoffice if you click on the user icon in the upper right corner,  emediatly after opening dialog it occurs an error. The same error occurs after viewing  Administrator user under Users. In DEV Tools calling function get2FAProvidersForUser?userId=-1 return error 500. Umbraco 9.5.0 fresh install is in question.

Pictures below:

![error-1](https://user-images.githubusercontent.com/73195407/167587380-19c38194-08a1-49ab-bbb5-2374cade6a10.PNG)
![error-2](https://user-images.githubusercontent.com/73195407/167587403-46ddac89-599c-4d32-8efb-a5feb88442b8.PNG)
![error-3](https://user-images.githubusercontent.com/73195407/167587425-8ad25eae-a96a-48b3-bb31-e96c128e2b67.PNG)
.

### Specifics

<html>
<body>

Category | Data
-- | --
Server OS | Microsoft Windows 10.0.19044
Server Framework | .NET 5.0.16
Default Language | hr
Umbraco Version | 9.5.0
Current Culture | hr-HR
Current UI Culture | hr-HR
Current Webserver | IIS
Models Builder Mode | InMemoryAuto
Debug Mode | True
Database Provider | System.Data.SqlServerCe.4.0
Browser | Chrome 101.0.4951.54
Browser OS | Win32

</body>
</html>

### Steps to reproduce

Croatian language file (hr.xml) you can download form my github repo to reproduce the error: 
Github: [https://github.com/alevak09/umbraco-v8-croatian-language](https://github.com/alevak09/umbraco-v8-croatian-language)

After copying the hr.xml inside the umbraco/config/lang/ , login to umbraco office, go under Users -> Administrator user,
then change the Language to: hrvatski (Hrvatska). After hitting CTRL + F5 you can see the error, or folowing the steps mentioned abowe in the bug summary. Did not find any error when changing language to other than croatian ie. french.

---
_This item has been added to our backlog [AB#19639](https://dev.azure.com/umbraco/243e7927-03b2-44e2-908f-d4ac7ea5daaa/_workitems/edit/19639)_

**Comment by bergmania** [comment]
Looks like it is an error related to NLS vs ICU 🤦

**Comment by p-m-j** [comment]
@alevak09 does your csproj have an itemgroup entry with the following

```xml
<RuntimeHostConfigurationOption 
    Condition="$(RuntimeIdentifier.StartsWith('linux')) Or $(RuntimeIdentifier.StartsWith('win')) Or ('$(RuntimeIdentifier)' == '' And !$([MSBuild]::IsOSPlatform('osx')))" 
    Include="System.Globalization.AppLocalIcu" 
    Value="68.2.0.9" />
  ```

**Comment by alevak09** [comment]
@bergmania I may be

@p-m-j Yes, it does have. Also found in NuGet package Manager installed package: Microsoft.ICU.ICU4C.Runtime

**Comment by bergmania** [comment]
PR here: https://github.com/umbraco/Umbraco-CMS/pull/12420

**Comment by nikolajlauridsen** [comment]
This has been fixed in #12420 and is headed for 9.5.1 😄
